src/c++/perf_analyzer/genai-perf/genai_perf/export_data/json_exporter.py

```diff
@@ -58,8 +58,9 @@ def export(self) -> None:
             f.write(json.dumps(self._stats_and_args, indent=2))
 
     def _prepare_args_for_export(self) -> None:
-        del self._args["func"]
-        del self._args["output_format"]
+        self._args.pop("func", None)
```

Comment:
these are functionally equivalent right?
why the change?

Reply:
`del` will throw a KeyError if key does not exist. `pop` instead will return default value if defined.

Reply:
well then this is better. THank you for making that change.